fix discord api, fix geolocation not show in post, fix display order of last update date

CGsama opened the pull request:
## Summary

<!--
  -
  - * Please describe your changes here *
  -
  - If you are going to resolve some issue, please add this context.
  - Resolve #ISSUE_NUMBER
  -
  - If you are going to fix some bug issue, please add this context.
  - Fix #ISSUE_NUMBER
  -
  -->
#850 fix discord api: discordapp.com -> discord.com
#851 fix geolocation not show in post
#852 fix display order of last update date: never post user as oldest update, was as latest update

CGsama committed:
change default theme

CGsama committed:
Update Feed Format by adding email

CGsama committed:
Update from discordapp.com to discord.app

CGsama committed:
update feed title to first 50 char of note content

CGsama committed:
Update feed.ts

CGsama committed:
Update feed.ts

CGsama committed:
Update feed.ts

CGsama committed:
Update feed.ts

CGsama committed:
Update feed.ts

CGsama committed:
Update feed.ts

CGsama committed:
Update feed.ts

CGsama committed:
Update feed.ts

CGsama committed:
Update feed.ts

CGsama committed:
switch to dinkiebitmap font

CGsama committed:
Update feed.ts

CGsama committed:
Update feed.ts

CGsama committed:
Update feed.ts

CGsama committed:
Update feed.ts

CGsama committed:
Update note.ts

CGsama committed:
Include renote and reply source

CGsama committed:
Bug fixing

CGsama committed:
Remove author info in leaf note & fix div class & fix code standard

CGsama committed:
Showusers order by updateAt NULL considered as max

mei23 commented:
Thank you for your contribution.

> #850 fix discord api: discordapp.com -> discord.com

Is there any reason not to use discordapp? 


> #853 enhance feed generation: include all attachments, and show discuss thread

I don't want to change this.
I want the following format in another repository: 
https://misskey.m544.net/@ia.rss
https://misskey.m544.net/@ia.atom
https://misskey.m544.net/@ia.json

CGsama commented:
Last year they were send an email says that will drop discordapp.com api on Nov 7, 2020.
Will undo the feed changes soon

```
Discord API Domain Migration

Last month, we excitedly announced our official move to discord.com. It was a long time in the making, and the work isn't done yet! For now, our API will continue to handle requests made to discordapp.com. On November 7, 2020 we will be dropping support for the discordapp.com domain in favor of discord.com. Please ensure that your libraries, bots, and applications are updated accordingly.

Due to technical constraints, our CDN domain will not be migrated and will remain cdn.discordapp.com for the foreseeable future.
```

CGsama committed:
fix discord api, geolocation in note, display order of last update date

fix discord api: discordapp.com -> discord.com
fix geolocation not show in post
fix display order of last update date: never post user as oldest update, was as latest update

mei23 commented:
@CGsama
Thanks a lot.
